### DISCUSSION TITLE
Music is not getting played, stucks and shows length 0:00 on linux mint.

### DISCUSSION THREAD

**Discussion by ghost** [KRTirtho/spotube#755] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

![image](https://github.com/KRTirtho/spotube/assets/134391156/d87b18cb-fb67-44ec-b8fc-46104bb48fdd)

### Expected Behavior

It should play and show the length of song.

### Steps to reproduce

I downloaded app using flatpak,
I opened app,
signed in my spotify account,
tried to play songs from my liked songs list,
It's not getting loaded as u can see in provided screenshot.

### Operating System

Linux Mint

### Spotube version

latest and v3.0.0 too

### Installation source

Flathub (Flatpak)

**Comment by KRTirtho** [comment, marked as answer]
Change "Settings" > "API Type" from "Piped" to "YouTube"
